src/BolWallet/ViewModels/AddMultiCitizenshipViewModel.cs

```diff
@@ -54,6 +70,20 @@ public async Task Generate()
         }
     }
 
+    public async Task CheckMultiCitizenship()
+    {
+        try
+        {
+            ShortHash = MultiCitizenshipShortHashModel.ShortHash;
+            IsMultiCitizenshipRegistered = await _bolService.IsMultiCitizenship(MultiCitizenshipShortHashModel.CountryCode, ShortHash);
+        }
+        catch (RpcException)
+        {
+            IsMultiCitizenshipRegistered = false;
+        }
+        catch (Exception) { }
```

Comment:
What is the expected behavior after this general exception gets caught and nothing happens?

Reply:
I removed it because our goal here is to catch the RpcException that indicates whether the shorthash is registered or not.